format.c

```diff
@@ -195,6 +205,8 @@ void format_like_time(Conf *conf)
       (long long) (user.tv_sec / conf->num_runs), (long long) ((user.tv_usec / 10000) / conf->num_runs));
 	fprintf(stderr, "sys  %9lld.%02lld\n",
       (long long) (sys.tv_sec / conf->num_runs), (long long) ((sys.tv_usec / 10000) / conf->num_runs));
+  fprintf(stderr, "cpu  %9lld.%02lld\n",
```

Comment:
This is a bit weird, I thought I had sublime set up to use 4-spaces as default. Anyhow, pushed a changset to fix this.